pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -344,6 +345,9 @@ public static class Broker {
     // precedence over "query.response.size" (i.e., "query.response.size" will be ignored).
     public static final String CONFIG_OF_MAX_SERVER_RESPONSE_SIZE_BYTES = "pinot.broker.max.server.response.size.bytes";
 
+    public static final String CONFIG_OF_NEW_SEGMENT_EXPIRATION_SECONDS = "pinot.broker.new.segment.expiration.seconds";
```

Comment:
nit: Can you add a JavaDoc for this variable to explain what this config does?

Reply:
you can also instead update this https://docs.pinot.apache.org/configuration-reference/broker

Reply:
Thanks! Will update pinot-docs once merged!